Mathlib/RingTheory/Polynomial/Basic.lean

```diff
@@ -786,10 +786,12 @@ theorem prime_C_iff : Prime (C r : MvPolynomial σ R) ↔ Prime r :=
       obtain ⟨s, a', b', rfl, rfl⟩ := exists_finset_rename₂ a b
       rw [← algebraMap_eq] at hd
       have : algebraMap R _ r ∣ a' * b' := by
-        convert killCompl Subtype.coe_injective |>.toRingHom.map_dvd hd <;> simp
+        convert _root_.map_dvd (killCompl Subtype.val_injective) hd
+        · simp only [algebraMap_eq, algHom_C]
+        · simp only [map_mul, killCompl_rename_app]
```

Comment:
Does `simp` not work here anymore? (I think all the arguments are already `@[simp]` tagged.)